src/pages/AdminList/useAdminList.ts

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect } from 'react';
+
+interface Admin {
+  id: number;
+  name: string;
+  email: string;
+}
+
+interface UseAdminListReturnType {
+  admins: Admin[];
+  searchQuery: string;
+  setSearchQuery: (query: string) => void;
+  page: number;
+  setPage: (page: number) => void;
+  itemsPerPage: number;
+  setItemsPerPage: (count: number) => void;
+  paginatedAdmins: Admin[];
+  pageCount: number;
+  startIndex: number;
+  endIndex: number;
+  getInitials: (name: string) => string;
+}
+
+const useAdminList = (): UseAdminListReturnType => {
+  const [admins] = useState<Admin[]>(
+    Array.from({ length: 50 }, (_, index) => ({
```

Comment:
add comment //TODO add real data when BE will be ready

Reply:
done